demo_app/app.py

```
@@ -0,0 +1,239 @@
+        url = f"https://tiles.afm.cibolabs.com/{subpath}"
+        # Fetch the image stream
+        response = session.get(url, headers={
+            "Authorization": f"Bearer {token}",
+            "Accept": "image/png"
+        }, stream=True)
+        
+        # Stream back to client
+        return Response(response.iter_content(chunk_size=1024), 
+                        status=response.status_code, 
+                        content_type=response.headers.get('Content-Type', 'image/png'))
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)